packages/core-forger/src/client.ts

```diff
@@ -37,93 +44,48 @@ export class Client {
 
     /**
      * Send the given block to the relay.
-     * @param  {(Block|Object)} block
-     * @return {Object}
      */
-    public async broadcast(block) {
+    public async broadcast(block: models.IBlockData): Promise<any> {
```

Comment:
Should add a struct/interface for the expected data we want to return instead of using `any`.